cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/ApplyLogic.java

```diff
@@ -88,4 +132,23 @@ private ConfigBean createConfigFromApplications(Map<String, ApplicationBean> app
         return applicationsConfigBean;
     }
 
+    /**
+     * @param serviceBeans the service beans that should be contained in the resulting config bean
+     * @return a config bean only containing the entered application beans
+     */
+    private ConfigBean createConfigFromServices(Map<String, ServiceBean> serviceBeans) {
+        SpecBean servicesSpecBean = new SpecBean();
+        servicesSpecBean.setServices(serviceBeans);
+        ConfigBean servicesConfigBean = new ConfigBean();
+        servicesConfigBean.setSpec(servicesSpecBean);
+        return servicesConfigBean;
+    }
+
+    public void setDiffLogic(DiffLogic diffLogic) {
```

Comment:
I understand that it makes the tests easier to write, but we don't opened our visibility for that in the class before. 
What do you think @cmatgh @M4eqP ?

Reply:
Not a big fan of it either. It'd be better to use dependency injection, as the class is instantiated for every single test anyway. But OTOH, this is not a public API...